setup.py

```diff
@@ -257,9 +267,26 @@ def check_linker_need_libatomic():
 if BUILD_WITH_SYSTEM_CARES:
     EXTENSION_LIBRARIES += ('cares',)
 
-DEFINE_MACROS = (('OPENSSL_NO_ASM', 1), ('_WIN32_WINNT', 0x600))
+DEFINE_MACROS = (('_WIN32_WINNT', 0x600),)
+asm_files = []
+NO_BORING_ASM = True
+if BUILD_WITH_BORING_SSL_ASM:
+   LINUX_X86_64 = "linux-x86_64"
+   if LINUX_X86_64 == util.get_platform():
```

Comment:
optional: You included .asm files for all architecture. Could it work across platforms? We can temporarily expand the allowed architectures and let the CI to find out for us.

Reply:
Included all architectures was the easiest thing to do.  I added what i think are appropriate cases for linux arm, mac osx 64 bit and windows 64 bit, we can see if they work.